hazelcast-jet-core/src/main/java/com/hazelcast/jet/Sources.java

```diff
@@ -388,6 +409,9 @@ private Sources() {
      * If the {@code ICache} is modified while being read, or if there is a
      * cluster topology change (triggering data migration), the source may
      * miss and/or duplicate some entries.
+     * <p>
+     * Default local parallelism for this processor is 2 (or less if less CPUs
```

Comment:
"or less if less CPUs are available" sounds contrived given that there is only one choice. "Or 1 if there's just one CPU available" sounds more natural.

Reply:
fixed